libs/community/langchain_community/embeddings/replicate.py

```diff
@@ -0,0 +1,72 @@
+import json
+from typing import Any, Dict, List, Optional
+
+from langchain_core.embeddings import Embeddings
+from langchain_core.pydantic_v1 import BaseModel, Extra, Field
+
+DEFAULT_MODEL_NAME = "replicate/all-mpnet-base-v2:b6b7585c9640cd7a9572c6e129c9549d79c9c31f0d3fdce7baac7c67ca38f305"
+
+
+class ReplicateEmbeddings(BaseModel, Embeddings):
+    """Replicate embedding models.
+
+    To use, you should have the ``replicate-python`` python package installed.
+
+    Example:
+        .. code-block:: python
+
+            from langchain_community.embeddings import ReplicateEmbeddings
+
+            model_name = "replicate/all-mpnet-base-v2:b6b7585c9640cd7a9572c6e129c9549d79c9c31f0d3fdce7baac7c67ca38f305"
+            re = ReplicateEmbeddings(
+                model_name=model_name
+            )
+    """
+
+    client: Any  #: :meta private:
```

Comment:
Add docstrings for all parameters

Reply:
i think for private params that aren't meant to be user specified fine to not document